pkg/log/zap/zap.go

```diff
@@ -207,3 +208,39 @@ func NewRaw(opts ...Opts) *zap.Logger {
 	log = log.WithOptions(o.ZapOpts...)
 	return log
 }
+
+// BindToFlagSet func allows CLI passed flags to be parsed.
+func (o *Options) BindFlags(fs *pflag.FlagSet) {
+	// Set Development mode value
+	fs.BoolVar(&o.Development, "zap-devel", false, "Enable zap development mode (changes defaults to console encoder, debug log level, and disables sampling)")
```

Comment:
`changes defaults to ...` We should also outline out what the default settings are if this is not set, i.e Production settings.